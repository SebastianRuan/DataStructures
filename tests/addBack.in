a 5 0
a 12 0
a 333 0
a 99 0
a 17 3
a 11 5
a 8 6
