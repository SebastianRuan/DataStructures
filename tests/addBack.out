99
333
12
17
5
11
8
